src/main/java/org/kuali/kfs/module/ar/businessobject/BillingPeriod.java

```diff
@@ -0,0 +1,116 @@
+package org.kuali.kfs.module.ar.businessobject;
```

Comment:
Since this is an overlay of a base KFS class, you may want to include the license header above the package declaration.